crates/rolldown/src/module_loader/runtime_normal_module_task.rs

```diff
@@ -112,6 +112,7 @@ impl RuntimeNormalModuleTask {
     );
     let mut symbol_for_module = AstSymbols::from_symbol_table(symbol_table);
     let facade_path = ResourceId::new("runtime");
+    let trivias = ast.with_mut(|fields| std::mem::take(fields.trivias));
```

Comment:
Probably, it's better to use it directly rather take out `trivias` field, it might be needed in the future. But this doesn't block merging PR.